mne_bids/dig.py

```diff
@@ -158,7 +158,7 @@ def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
     fname : str
         Filename to save the electrodes.tsv to.
     kind : str
-        Type of the data as in ALLOWED_KINDS. For iEEG, requires size.
+        Type of the data as in ALLOWED_MODALITY_KINDS. For iEEG, requires size.
```

Comment:
How would users know how to find this mysterious `ALLOWED_MODALITY_KINDS` thing?

Reply:
What would be the correct fix? 

I guess we could:
- create a link to it? Is that using the "``" double characters?
- explicitly type it out? This could be annoying to maintain since that list might get larger?

Reply:
Maybe it's time to switch to the `docdict` approach MNE-Python uses…

Reply:
:( don't know how to do that but for now I can link in the exact strings.